build-android-x86.sh

```
@@ -1,13 +1,26 @@
+# If BUILD_SHARED_LIBS is OFF, then libonnxruntime.a is linked into libsherpa-onnx-jni.so
+# and you only need to copy libsherpa-onnx-jni.so to your Android projects.
+#
+# If BUILD_SHARED_LIBS is ON, then you need to copy both libsherpa-onnx-jni.so
+# and libonnxruntime.so to your Android projects.
+#
+if [ -z $BUILD_SHARED_LIBS ]; then
+  BUILD_SHARED_LIBS=ON
 fi
 
-dir=$PWD/build-android-x86
+if [ $BUILD_SHARED_LIBS == ON ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)